package.json

```diff
@@ -5,17 +5,23 @@
   "main": "index.js",
   "scripts": {
     "build": "tsc",
+    "clean": "rm -rf ./dist && rm -rf ./node_modules",
+    "dev": "./node_modules/.bin/nodemon ./src/index.ts",
     "start": "node ./dist/index.js",
     "test": "vitest run",
-    "lint": "rome check src"
+    "lint": "rome check src",
+    "preinstall": "if [ ! -f \"/is-docker\" ]; then echo \"This script must be run inside our Docker container\" >&2; npm run clean; exit 1; fi"
   },
   "author": "",
   "license": "ISC",
   "devDependencies": {
     "@types/express": "^4.17.21",
     "@types/node": "^20.11.27",
     "@types/supertest": "^6.0.2",
+    "bcrypt": "^5.1.1",
```

Comment:
bcrypt will compile a binary post install.. it would explode if there is a platform mismatch from host to container.